spec/setup.spec.js

```diff
@@ -44,6 +48,12 @@ describe('Setup/Destroy TestCase', function () {
             expect(document.querySelector('.medium-editor-toolbar')).toBeTruthy();
             editor.destroy();
             expect(document.querySelector('.medium-editor-toolbar')).toBeFalsy();
+
+            expect(document.querySelector('[data-medium-editor-element]')).toBeFalsy();
+            expect(document.querySelector('[data-medium-focused]')).toBeFalsy();
+            expect(document.querySelector('[aria-multiline]')).toBeFalsy();
+            expect(document.querySelector('[medium-editor-index]')).toBeFalsy();
+            expect(document.querySelector('[role]')).toBeFalsy();
```

Comment:
If we did this check here, we could ensure we don't add any other stray attributes in the future:

```js
expect(this.el.getAttribute('class')).toBe('editor');
expect(this.el.attributes.length).toBe(1); // 'class' should be the only remaining attribute
```